### DISCUSSION TITLE
Question.

### DISCUSSION THREAD

**Discussion by WarpWing** [mmtrt/leagueoflegends#174] [Q&A]
Hey all! Can someone tell how to install blitz.gg for Linux? I know there was some sort of install you could do alongside this snap if I remember correctly. Can anyone help me?

**Comment by mmtrt** [comment]
Due to new version updates of blitzgg requiring recent .NET framework to work. New .NET framework has issues on old wine version and league patches are not ported to recent wine version which also have fixes for .NET framework.

**Reply by WarpWing** [reply]
I see, is there a ETA as to when the .NET Fix comes out because blitz.gg is nice

**Reply by mmtrt** [reply]
It depends when upstream WINE have league fixes to run it without patching then I'll update league WINE snaps so it's waiting game at the moment.
